api/libproject/devstart.rb

```diff
@@ -478,21 +478,30 @@ def run_gradle(cmd_name, args)
   :fn => ->(*args) { run_gradle("gradle", args) }
 })
 
+def connect_to_db(*args)
+  if args.empty?
```

Comment:
This breaks as soon as you add a second flag, here is the preferred way to set a default: https://github.com/all-of-us/workbench/blob/0ca9ec7874214717a3e083b9c4c5b4faaf76c729/api/libproject/devstart.rb#L1042

Reply:
thanks